README.md

```diff
@@ -1 +1,121 @@
-# js-todo-list
\ No newline at end of file
+# Javascript To-do List
+
+> This project is about building a static website for a restaurant using pure javascript, Webpack, modules, building the entire site using just Javascript, and get more practice on DOM manipulation by dynamically rendering a simple restaurant homepage.
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+
+- HTML
+- Javascript
+- Webpack
+- Webpack modules
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/MiguelDP4/js-todo-list/feature/dist/index.html)
+
+
+## Getting Started
+
+**Want to check the code and make some changes please .**
+
+Do you want to clone this project please use this command on you terminal
+
+```
+git clone https://github.com/MiguelDP4/js-todo-list.git
+```
+
+webpack is used to compile JavaScript modules. Once installed, you can interface with webpack either from its CLI or API. If you're still new to webpack, please read through the core concepts and this comparison to learn why you might use it over the other tools that are out in the community.
+
+Core concepts : [Webpack](https://webpack.js.org/concepts/)
+
+
+### Prerequisites
+
+- Webpack 
+- NPM 
+
+### Setup
+
+Install NPM
+
+Check that you have node and npm installed
+
+To check if you have Node.js installed, run this command in your terminal:
+
+
+```
+node -v
+```
+
+To confirm that you have npm installed you can run this command in your terminal:
+
+
+```
+npm -v
+```
+
+
+### Install
+
+install npm and webpack
+
+
+```
+sudo apt update
+sudo apt install npm
+```
+
+Local Installation Webpack
+
+To install the latest release or a specific version, run one of the following commands:
+
+```
+npm install --save-dev webpack
+npm install --save-dev webpack-cli
+```
+
+Global Installation Webpack
+
+The following NPM installation will make webpack available globally:
+
+```
+npm install --global webpack
+```
+
+You can then compile the code with the next command
+
+```
+npm start
+```
+After this, a main.js file will be created and written in the dist folder. Copy and paste index.html and main.js wherever you want and you can now see them in your browser.
```

Comment:
`Copy and paste ... your browser` I don't think it's the good way to start the application.
Find a simple way to do it.

Also I suggest you install and set up `webpack-dev-server` which easy the launch of the application.

Reply:
solved